articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+![Ring Topology](/engineering-education/network-types-topologies/ring.jpg)
+
+#### Mesh Topology
+Mesh topology is the solution to the problem with ring topology. Unlike ring topology, here *every node is connected with every other node present in the network*. In the case of a single node failure, the rest of the system can still be functional. Since one node connects many other nodes, data is shared through any one of the many paths available. This makes mesh arrangement a suitable network for industries and business establishments. A mesh topology in which all the nodes are connected to each other is called full connectivity, and the mesh topology in which only some nodes are connected to each other is called partial connectivity.
+Mesh topology increases the network flow and data can be shared faster. It also records a lesser number of network breakages and system breakdowns. But since all the nodes are connected to each other, each node low key acts as a router to another node and so on. This increases power consumption to a larger extent. Mesh topology needs lots of proper planning and requires high maintenance. Due to many links, high network latency may be an issue.
+
+![Mesh Topology](/engineering-education/network-types-topologies/mesh.jpg)
+
+#### Star Topology
+*This topology consists of an extra part, i.e. hub or switch, in the center of the topology which connects all the nodes*. A hub or switch is a networking device that transfers data from one device to another device. It acts as a transmission medium. While both are used depending on their operations, some [differences](https://www.geeksforgeeks.org/difference-between-hub-and-switch/) may affect the way they perform. This is the most commonly used topology.
+In an emergency of network failure, identifying the damaged node becomes easier. This is because a single hub connects all the nodes. Here, the damaged node will be shut down and the rest of the nodes can still be functional. But if a hub is damaged then all the nodes connected to that hub will be shut down for reconstruction.
+
+![Star Topology](/engineering-education/network-types-topologies/star.jpg)
+
+#### Tree Topology
+Tree topology is a special kind of bus topology. Here *central nodes or central hubs are connected to the main bus*. These nodes or hubs are also called as **root nodes**. The other peripheral nodes are connected to those nodes or hubs. These peripheral nodes are called as **descendant nodes**. Like the bus topology, terminators are attached at both ends to define the length of the main bus. This type of topology is used where the data has to be shared in a hierarchical manner.

```

Comment:
"shared in a hierarchical manner." 
Perhaps an example

Reply:
Provided examples